spec/controllers/sign_up/registrations_controller_spec.rb

```diff
@@ -21,7 +21,7 @@
       expect(response).to redirect_to account_path
     end
 
-    it 'gracefully handles invalid formats' do
+    xit 'gracefully handles invalid formats' do
```

Comment:
what would it take to bring this spec back? I think it was guarding for real regressions we might see from various scans

Reply:
that was my hope as well. it looks like we can render a 406 error page, but that's still a significant change in behavior. do we do scans that send malformed Accept or Content-Type headers and need the server response to ignore that?

Reply:
The goal of the spec is to not blow up, so anything other than a 500 should be acceptable

Reply:
added in 471e5e2 and cbac137